sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -628,6 +628,17 @@ def test_callable_k(self):
       with_keys = pc | util.WithKeys(lambda x: x * x)
     assert_that(with_keys, equal_to([(1, 1), (4, 2), (9, 3)]))
 
+  @staticmethod
+  def _test_args_kwargs_fn(x, multiply, subtract):
+    return x * multiply - subtract
+
+  def test_args_kwargs_k(self):
```

Comment:
Could you add a test case with fn that does not take side inputs?

Reply:
The previous test cases test `WithKeys` for constant and callable functions, so those cases should already be covered.